fairseq/criterions/label_smoothed_cross_entropy_with_alignment.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) 2017-present, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed under the license found in the LICENSE file in
+# the root directory of this source tree. An additional grant of patent rights
+# can be found in the PATENTS file in the same directory.
```

Comment:
nit: please update to the new header: https://github.com/pytorch/fairseq/blob/master/fairseq/criterions/label_smoothed_cross_entropy.py#L1-L4

Reply:
Done.